examples/05_batch_gemm/batch_gemm.hpp

```diff
@@ -173,8 +173,8 @@ class batch_gemm_t {
     /// @return The size of local memory required.
     __XETLA_API static constexpr uint32_t get_slm_size() {
         constexpr uint32_t size = gemm_t::slm_size + epilogue_t::slm_size;
-        static_assert(size <= (128 * 1024),
-                "The local memory size should be less than 128KB!");
+        static_assert(size <= arch_attr_t<arch_tag>::local_mem_size,
+                "The local memory size excess!");
         return size;
```

Comment:
I don't think so. `static_assert ` can only have constant error message until c++26
See [static_assert declaration (since C++11) - cppreference.com](https://en.cppreference.com/w/cpp/language/static_assert)